src/styles/components/_home-search-box.scss

```diff
@@ -1,15 +1,73 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  width: 80%;
+
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px 0px) hsla(0, 0%, 0%, 0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
+
+  & > input {
+    width: 75%;
+    padding: rem(5px 10px 5px 10px);
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+    text-align: left;
+
+    @include respond-above('sm') {
+      padding: rem(10px 20px 10px 20px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      padding: rem(20px 40px 20px 40px);
+      font-size: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(35px 60px 35px 60px);
+      font-size: rem(24px);
+    }
+  }
 
   &__button {
-    @include button-darken-effect($picton-blue);
+    position: absolute;
+    top: 10%;
+    right: 15%;
+
+    width: 5%;
+    height: 80%;
+    padding: 0%;
+
+    background-color: $light;
+    border: none;
+    outline: none;
```

Comment:
Please don't remove outline. See [this](https://stackoverflow.com/questions/20340138/remove-blue-border-from-css-custom-styled-button-in-chrome)